learningresources/models.py

```diff
@@ -5,6 +5,14 @@
 from __future__ import unicode_literals
 
 import logging
+try:
```

Comment:
Replace this with [six.moves.urllibparse](https://pythonhosted.org/six/#module-six.moves.urllib.parse), I think we should have it from Django, but it isn't explicit in our requirements file, so maybe add it there too.